src/app/content/components/ContentExcerpt.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import { bodyCopyRegularStyle } from '../../components/Typography';
+import addTargetBlankToLinks from '../utils/addTargetBlankToLinks';
+
+interface Props {
+  content: string;
+}
+
+// tslint:disable-next-line:variable-name
+const ContentExcerpt = styled((props: Props) => {
+  const {
+    content,
+    ...excerptProps
+  } = props;
+
+  const fixedContent = React.useMemo(
+    () => addTargetBlankToLinks(props.content),
+    [props.content]
+  );
+
+  return <div
+    className='content-excerpt'
```

Comment:
youre losing this to the classname that styled is passing in

Reply:
fixed